resources/js/processes/modeler/initialLoad.js

```diff
@@ -465,4 +465,17 @@ ProcessMaker.EventBus.$on(
   },
 );
 
+ProcessMaker.EventBus.$on(
+  "modeler-init",
+  (evento) => {
+    evento.registerPreview({
+      url:'/designer/screens/preview',
+      receivingParams: ['screenRef'],
+      matcher: (nodeData) => {
+        return nodeData.$type && nodeData.$type === 'bpmn:Task'
```

Comment:
try to see if this is compiled
```suggestion
        return nodeData?.$type === 'bpmn:Task'
```